glossary_5b2d37510428632c466b3c8c.md

```diff
@@ -67,9 +67,6 @@
   pipeline run.
 ^
 
-* repository: A GitHub repository 
```

Comment:
@mactsouk please restore this for now. Keep it as:

`repository: TODO`

Reply:
Done!

Reply:
@mactsouk Thanks! Feel free to merge 🚢 !